GoogleForCreators/web-stories-wp
# Quick Actions: FTUE Integration

## Context
Part of Quick Actions work.

## Summary
This PR triggers relevant tips for fg and bg media when using the media replace quick action

## Relevant Technical Choices
NA

## To-do



## User-facing changes
NA
-------------------------------
(This is all behind the quick actions experiment)
Triggers tips when FG Media Swap or BG Media Swap are clicked

https://user-images.githubusercontent.com/35983235/120345371-09de3900-c2c0-11eb-9cbd-62ba383573ae.mp4


https://user-images.githubusercontent.com/35983235/120345373-0a76cf80-c2c0-11eb-8776-60b75970a3e7.mp4




## Testing Instructions
enter this into your terminal to clear your read quick tips:
```
await wp.apiFetch({ path: '/web-stories/v1/users/me', method: 'POST', data: { meta: { web_stories_onboarding: {} }}})
```

Then add a bg image or fg image and click the replace media quick action. Should trigger a relevant ftue tip


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7684
